sorc/checkout.sh

```diff
@@ -72,11 +75,10 @@ fi
 echo EMC_post checkout ...
 if [[ ! -d gfs_post.fd ]] ; then
     rm -f ${topdir}/checkout-gfs_post.log
-    git clone https://github.com/NOAA-EMC/EMC_post.git gfs_post.fd >> ${topdir}/checkout-gfs_post.log 2>&1
+    git clone https://github.com/NOAA-EMC/EMC_post.git gfs_post.fd >> ${logdir}/checkout-gfs_post.log 2>&1
     cd gfs_post.fd
-    git checkout ff42e0227d6100285d4179a2572b700fd5a959cb
+    git checkout upp_v10.0.3
```

Comment:
https://github.com/NOAA-EMC/EMC_post/commit/ff42e0227d6100285d4179a2572b700fd5a959cb is from April where as the upp_v10.0.3 is from January.  Should we use the more recent UPP hash?

Reply:
If the coupled system doesn't specifically need the upp_v10.0.3 tag then I agree that the newer hash should remain. That's the hash that was tested with the hpc-stack updates commit.

Reply:
Should we just use the latest UPP tag here?

Reply:
the atmosphere only does inline post, so I don't think that will break @JianKuang-UMD's branch.

Reply:
module_base is loading upp/10.0.6, so I propose we check out that version

Reply:
Alternatively, UPP is now up to 10.0.8. I can update all of them to that tag

Reply:
sure.  I don't really have a preference.  It is used for offline post processing, and who is using that now?

Reply:
I think we might be using some of the scripts? But I don't think the version really matters all that much.

Reply:
Updated all to 10.0.8 for consistency